mne/io/bti/bti.py

```diff
@@ -1116,13 +1129,19 @@ def _get_bti_info(pdf_fname, config_fname, head_shape_fname, rotation_x,
     use_hpi = False  # hard coded, but marked as later option.
     logger.info('Creating Neuromag info structure ...')
     info = _empty_info()
-    date = bti_info['processes'][0]['timestamp']
-    info['meas_date'] = [date, 0]
-    info['sfreq'] = 1e3 / bti_info['sample_period'] * 1e-3
+    if pdf_fname:
```

Comment:
`is not None`

In general it's better to have `is not None` -- more explicit, and saves potential headaches in case we add some behavior where it can also be `False`, `[]`, `''`, or many other potential things that evaluate to `False` on bool eval.